### DISCUSSION TITLE
C# script does not have a valid MCCScript header

### DISCUSSION THREAD

**Discussion by h4rmy** [ORelio/Minecraft-Console-Client#1742] [Q&A] [answered]
Im having trouble running c# scripts that are listed in the repository, is this a bug no one talking about?
Im using the latest build #523
![image](https://user-images.githubusercontent.com/75831703/130793773-8ed5bf94-2a12-4b0b-ae99-1a912120f383.png)

**Comment by ReinforceZwei** [comment, marked as answer]
Hi,

The script you are loading is a built-in bot, means that you do not need to load the script via `/script` command. To enable `AutoRespond`, edit your `MinecraftClient.ini` file and set `enable=true`.
https://github.com/ORelio/Minecraft-Console-Client/blob/41b0325f45b3084ae66ced37e9311927360d6aa8/MinecraftClient/Resources/config/MinecraftClient.ini#L134-L136
You may also want to see the [User Manual](https://github.com/ORelio/Minecraft-Console-Client/tree/master/MinecraftClient/config) and [`AutoRespond` example usage](https://github.com/ORelio/Minecraft-Console-Client/blob/master/MinecraftClient/config/sample-matches.ini)

**Reply by h4rmy** [reply]
This is the solution for my problem, but i do have more issues with using that on servers. Could you help me on that? Theres my telegram @h4rmy
![image](https://user-images.githubusercontent.com/75831703/130800724-992740d5-8d8e-4531-9fd7-8e04306d9e1a.png)

**Reply by ReinforceZwei** [reply]
Seems like your server has custom chat format. See [detecting chat messages](https://github.com/ORelio/Minecraft-Console-Client/tree/master/MinecraftClient/config#detecting-chat-messages).
